app/assets/stylesheets/application/style_guide.scss

```diff
@@ -1,9 +1,38 @@
-#pages-style-guide {
+#style-guide-style-guide {
   margin: 0 auto;
   max-width: 960px;
+
+  [class^=outer-container-]:nth-of-type(even) {
+    background-color: lighten(rgba(255, 0, 0, .75), 45%);
+  }
+
+  [class^=outer-container-]:nth-of-type(odd) {
+    background-color: lighten(rgba(255, 0, 0, .75), 40%);
+  }
+
+  [class^=row-]:nth-of-type(even) {
+    background-color: lighten(rgba(0, 0, 255, .75), 45%);
+  }
+
+  [class^=row-]:nth-of-type(odd) {
+    background-color: lighten(rgba(0, 0, 255, .75), 40%);
+  }
+}
+
+.style-guide-navigation {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.